### PR TITLE
Make getting-started/stats

### LINKED ISSUE #1773: New docs section: Getting Started > Statistical Modeling
- Generally a home for sk-learn-ish topics
- Move the PCA docs here from the User Guide
- Maybe add a point to the regression notebook, but not 100% sure about this

### PR DESCRIPTION
**Pull request by mccalluc**
- Fix #1773
- Surprised PCA is missing in reference: Just filed #1798 
- #1770 might be referenced here

### CODE REVIEW

File: docs/source/getting-started/stats/index.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Statistical Modelling
```
Comment (Shoeboxam) [inline docs/source/getting-started/stats/index.rst:1]: Can we use US English (one "l")? Sorry UK folks!

File: docs/source/getting-started/stats/index.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+=====================
+
+Beyond simple means and medians, a range of statistical techniques have DP variants.
+Some, like linear regression, can be built with the tools we've already introduced.
+Others, like PCA, are supported under the ``opendp.sklearn`` namespace.
```
Comment (Shoeboxam) [inline docs/source/getting-started/stats/index.rst:6]: I would say that we intend to provide more functionality in this space in the future and are accepting contributions. For the time being, we have PCA and consider it a model for future functionality.

File: docs/source/getting-started/stats/pca.ipynb
Comment (Shoeboxam) [inline docs/source/getting-started/stats/pca.ipynb:1]: Did this get converted? Might need to be rebased.

**Review by Shoeboxam — CHANGES_REQUESTED** [review]
Minor comments.

File: docs/source/getting-started/stats/index.rst (hunk 1 above)
Comment (mccalluc) [inline docs/source/getting-started/stats/index.rst:1]:
```suggestion
Statistical Modeling
```

File: docs/source/getting-started/stats/index.rst (hunk 2 above)
Comment (mccalluc) [inline docs/source/getting-started/stats/index.rst:6]:
```suggestion
We intend to provide more functionality in this space in the future and are accepting contributions.
For the time being, OpenDP has PCA and we consider it a model for future functionality.
```

**Review by mccalluc — COMMENTED** [review]
updated text

**Review by Shoeboxam — APPROVED** [review]
Thanks!

**Pull request merged**